src/GrammarApplicator_runGrammar.cpp

```diff
@@ -621,7 +635,7 @@ void GrammarApplicator::runGrammarOnText(std::istream& input, std::ostream& outp
 
 				if (line[0]) {
 					if (lSWindow && lCohort && testStringAgainst(line, text_delimiters)) {
-						lSWindow->text_post += &line[0];
+						lSWindow->text_post += line;
```

Comment:
No, this and others must be `&line[0]`. `line` is used as a buffer, so `line.size()` is much bigger than the relevant null-terminated content.